assets/model_monitoring/components/model_monitor/model_monitor_output_signal_metrics/spec.yaml

```diff
@@ -0,0 +1,52 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_monitor_output_signal_metrics
+display_name: Model Monitor - Output Signal Metrics
+description: Output the computed model monitor metrics.
+version: 0.3.2
+is_deterministic: true
+
+code: ../../src/
+entry:
+  file: ./model_monitor_output_signal_metrics/run.py
+
+inputs:
+  signal_metrics: 
```

Comment:
nit: Since the component is named 'output signal metrics' it is a little confusing that signal metrics is the input.

How about we name the component 'Model Monitor - Metrics Output Builder'? For common components used by all signals, we have not used the term 'signal' anywhere.

Inputs can be 'computed_metrics' (since the components are called compute metrics), 'signal name' and 'signal type' and output would be 'metrics_output'.

What do you think?

Reply:
Ah maybe keep the input as 'signal_metrics' since we use that term in the prev component - avoid breaking changes :)

Reply:
That is a good point. I renamed the component to 'Model Monitor - Metrics Outputter' and left the name of object builder class unchanged. Also left the `signal_metrics` input name unchanged so that we don't confuse ourselves. If needed, we can rename it post-cutover.